### PR TITLE
fix: Fix color attribute shared styles

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/views/TradingCompetition/components/CompetitionHeadingText.tsx [hunk 1]
```diff
@@ -21,8 +21,8 @@ const H2SizeStyles = (theme: DefaultTheme) => `
   }
 `
 
-const sharedStyles = (props: HeadingProps) => `
-  color: ${props.textColor ? props.textColor : '#ffffff'};
+const sharedStyles = (props) => `
+  color: ${props.textColor ? props.theme.colors[props.textColor] : '#ffffff'};
```
Comment (ChefHutch) [inline src/views/TradingCompetition/components/CompetitionHeadingText.tsx:25]: I'm not sure any implementations of the `Heading1Text` / `Heading2Text` components even make use of this prop, I can probably rip this out. Will look tomorrow during trading comp code cleanup

Comment (hachiojidev) [reply src/views/TradingCompetition/components/CompetitionHeadingText.tsx]: Please add the typing back unless Typescript can correctly infer the props.

**Review by hachiojidev — CHANGES_REQUESTED** [review]
@ChefHutch Let me know if this PR is okay.

File: src/views/TradingCompetition/components/CompetitionHeadingText.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/TradingCompetition/components/CompetitionHeadingText.tsx]: Done

File: src/views/TradingCompetition/components/CompetitionHeadingText.tsx [hunk 2]
```diff
@@ -22,7 +22,6 @@ const H2SizeStyles = (theme: DefaultTheme) => `
 `
 
 const sharedStyles = (props: HeadingProps) => `
-  color: ${props.textColor ? props.textColor : '#ffffff'};
```
Comment (ChefHutch) [inline src/views/TradingCompetition/components/CompetitionHeadingText.tsx:25]: We still need this color to be assigned, but the prop is OK to be removed, so just:

```css
color: #ffffff;
```

if removed entirely:
Screenshot 2021-04-09 at 15 02 08

**Review by ChefHutch — CHANGES_REQUESTED** [review]

File: src/views/TradingCompetition/components/CompetitionHeadingText.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/TradingCompetition/components/CompetitionHeadingText.tsx:25]: Done

**Review by ChefHutch — APPROVED** [review]

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**